core_backend/app/db/db_models.py

```diff
@@ -449,6 +449,20 @@ async def get_similar_content(
     return await get_search_results(asession, question_embedding, n_similar)
 
 
+async def get_similar_content_async(
+    asession: AsyncSession,
+    question: UserQueryBase,
+    n_similar: int,
+) -> Dict[int, UserQuerySearchResult]:
```

Comment:
nit: should we put asession as the last argument? To be consistent with the functions in the question_answer router?

Reply:
Actually there is not consistency accross the codebase. In `db_models.py` `asession` starts first while in 
most routers,  `asession` is last. Let's agreed on what to use. And I will update in next PR.